README.md

```diff
@@ -12,7 +12,11 @@
 | daily N                 | Gives all india daily changes analysis for past `N` days    | :heavy_check_mark: | `daily 15`     |
 | test State              | Gives all the test centers for Corona in the given state    | :heavy_check_mark: | `test punjab`  |
 | new                     | Gives all the states with sorted new cases                  | :heavy_check_mark: | `new`          |
+<<<<<<< HEAD
```

Comment:
You can remove these unwanted arrows and HEAD.

Reply:
Okay will do it, didn't knew they were there 😬